css-three-column-layout/styles.css

```diff
@@ -0,0 +1,47 @@
+body {
+  font-family: Lato, sans-serif;
+  background-color: #1896da;
+}
+
+.container {
+  width: 900px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
```

Comment:
You should make this into a separate class. Remember, the only styling that should be on the `row` class is `display: flex`